Cyclismo/libs/JTurbo/src/org/cowboycoders/turbotrainers/bushido/brake/ConstantResistanceController.java

```diff
@@ -1,13 +1,17 @@
 package org.cowboycoders.turbotrainers.bushido.brake;
 
+import org.cowboycoders.turbotrainers.Mode;
 import org.fluxoid.utils.SimpleCsvLogger;
 
 import java.io.File;
 
 public class ConstantResistanceController extends AbstractController {
 
+  // Doesn't support any of the current modes
+  private static final Mode SUPPORTED_MODE = null;
+
 	private SimpleCsvLogger logger;
-	private int resistance = 0;
+	private int resistance = 100;
```

Comment:
I changed the default resistance to the same value used during calibration, so it doesn't need to be set during calibration. I can change it back so that we override the default mapper in the calibration.